pkg/machines/components/storagePools/storagePool.jsx

```diff
@@ -128,6 +128,13 @@ StoragePool.propTypes = {
 };
 
 class StoragePoolActions extends React.Component {
+    onActivate() {
+        storagePoolStart(this.props.storagePool.connectionName, this.props.storagePool.id)
```

Comment:
Let's please use a consistent naming. If the libvirt terminology is Activate/Deactivate, then use that everywhere, othwise Start/Stop.

Reply:
virsh commands are ```virsh pool-start```` and ```virsh pool-destroy````.
The API exposes the virStoragePoolCreate and virStoragePoolDestroy function for the above.
And then the libvirt documentation uses active/inactive storage pool to refer to their state.
So there is some mess with naming from libvirt side already :(

But yes, let's keep the function names at least consistent, I think I prefer activate/deactivate but I am open to suggestions.